CMakeLists.txt

```diff
@@ -10,8 +10,9 @@ include(cmake/install-prefix.cmake)
 include(cmake/wsl.cmake)
 include(cmake/simd.cmake)
 
-set(CMAKE_C_STANDARD 11)
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
This is a minor change. Modifying the flag sets the language standard (e.g. -std=c++11 rather than -std=gnu++11) to increase compatibility across compilers.